maintenance/migratestatus/migratestatus.go

```diff
@@ -75,6 +76,13 @@ func (o *options) Validate() error {
 		return errors.New("'--dest' is required unless using '--retire' mode.\n")
 	}
 
+	if o.descriptionURL != "" {
+		_, err := url.ParseRequestURI(o.descriptionURL)
+		if err != nil {
+			return errors.New("'--description' URL is not valid\n")
```

Comment:
nit: we should propagate the `err` out to better inform what went wrong

Reply:
Good point.